zeroize/tests/zeroize_derive.rs

```diff
@@ -285,4 +285,49 @@ mod custom_derive_tests {
         }
         assert_eq!(&value.0 .0, &[0, 0, 0])
     }
+
+    #[test]
+    fn check_actual_memory_vec() {
```

Comment:
Nit: there's no `Vec` involved here. Perhaps:

```suggestion
    fn check_actual_memory_array() {
```